common/config/webpack/sampleapp.webpack.config.js

```diff
@@ -76,7 +76,8 @@ const webpackConfig = (sampleAppDir, env, babelConfig) => {
       new CopyPlugin({
         patterns: [
           { from: path.resolve(sampleAppDir, "public/manifest.json"), to: "manifest.json" },
-          { from: path.resolve(sampleAppDir, "public/assets"), to: "assets",  noErrorOnMissing: true }
```

Comment:
this looks like an incorrect merge conflict resolve - revert changes to this file, its since been updated to copy all assets in public/assets and not need to specify sub-folders

Reply:
I've reversed it. Thanks.